tests/common/deflate.cpp

```diff
@@ -87,6 +87,22 @@ GTEST_TEST(DEFLATE, decompressBuf) {
 	delete[] decompressed;
 }
 
+GTEST_TEST(DEFLATE, decompressBufWithoutDecompressedSize) {
+	static const size_t kSizeCompressed   = sizeof(kDataCompressed);
+	static const size_t kSizeDecompressed = strlen(kDataUncompressed);
+
+	size_t size = 0;
+	const byte *decompressed =
+		Common::decompressDeflateWithoutOutputSize(kDataCompressed, kSizeCompressed, size, Common::kWindowBitsMaxRaw);
```

Comment:
Could you give it a low frameSize, so that it does at least do two iterations of that extract loop? Otherwise, we're not testing that this is actually working correctly.